spec/controllers/api/v1/froggo_teams_controller_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::FroggoTeamsController, type: :controller do
+  let(:user) { create(:github_user) }
+  let(:platanus_org) { create(:organization, login: "platanus", members: [user]) }
+  let!(:platanus_team) { create(:froggo_team, name: "test_name", organization: platanus_org) }
+  let(:buda_org) { create(:organization) }
+  let!(:buda_team) { create(:froggo_team, name: "test_name", organization: buda_org) }
+  let(:github_session) do
+    double(
+      token: "token",
+      organizations: [{ id: platanus_org.gh_id, role: "member" }],
+      user: user
+    )
+  end
+
+  before do
+    expect(subject).to receive(:authenticate_github_user).and_return(true)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/MessageSpies: Prefer have_received for setting message expectations. Setup subject as a spy using allow or instance_spy.